test/common/src/main/kotlin/org/codice/compliance/verification/core/RequestProtocolVerifier.kt

```diff
@@ -70,37 +80,48 @@ class RequestProtocolVerifier(val request: Node) {
                     && request.children("Assertion")
                             .map { it.children("AuthnStatement") }
                             .filter { it.isNotEmpty() }
-                            .any { it.none { it.attributes.getNamedItem("SessionIndex")?.textContent == querySessionIndex } })
-                throw SAMLComplianceException.create(SAMLCore_3_3_2_2_a, message = "There was no AuthnStatement in the Assertion that had a SessionsIndex of $querySessionIndex.")
+                            .any {
+                                it.none {
+                                    it.attributes.getNamedItem("SessionIndex")?.textContent == querySessionIndex
+                                }
+                            })
+                throw SAMLComplianceException.create(SAMLCore_3_3_2_2_a,
```

Comment:
✏️ Just noticed this: Should we put this exception in curly brackets inside the if statement?

Reply:
Yes. Missed that.